plugins/search/providers/elastic_search/lib/model/kESearchQueryManager.php

```diff
@@ -77,7 +89,12 @@ public static function getExactMatchQuery($searchItem, $fieldName, $allowedSearc
 			$queryType = self::MATCH_PHRASE_KEY;
 
 		$searchTerm = elasticSearchUtils::formatSearchTerm($searchItem->getSearchTerm());
-		$exactMatch[$queryType] = array( $fieldName . $fieldSuffix => $searchTerm);
+		$fieldBoostFactor = $searchItem::getFieldBoostFactor($fieldName);
+		$exactMatch[$queryType] = array
+			( $fieldName . $fieldSuffix => array
+				(self::QUERY_KEY => $searchTerm, self::BOOST_KEY => $fieldBoostFactor)
```

Comment:
also need to handle the case of term query. 
(for term query use VALUE_KEY, for match phrase use QUERY_KEY)